search-resources-dynamodb-trigger/src/test/java/no/unit/nva/publication/PublicationUpdateEventHandlerTest.java

```diff
@@ -149,7 +151,7 @@ public void handlerIndexesDocumentInEsWhenEventIsValidUpdate()
         assertThat(indexedDocument, is(equalTo(expectedDocument)));
     }
 
-    @Test
+    // @Test
```

Comment:
There is a `@Disabled` tag. better this or delete the test

Reply:
Test is enabled again, was an error in getType() for IndexDocument